src/sfctl/custom_cluster.py

```diff
@@ -39,6 +39,16 @@ def select_arg_verify(endpoint, cert, key, pem, ca, aad, no_verify):
     if pem and any([cert, key]):
         raise CLIError(usage)
 
+def show():
+    """
```

Comment:
Can you add the help string to the helps files, just like `sfctl cluster select` has.

Reply:
ok..will update the help with a new PR

Reply:
might need some more information for this. which file you are suggesting to update?

Reply:
You need to add a new file or update the `main.py` file in `service-fabric-cli/src/sfctl/helps/` to reflect your new command. Using a doctstring in the function signature is against our convention.